ivy_tests/test_ivy/test_frontends/test_torch/test_spectral_ops.py

```diff
@@ -0,0 +1,35 @@
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.test_frontends.test_torch.test_tensor import _requires_grad
+from hypothesis import strategies as st
+from ivy_tests.test_ivy.helpers.testing_helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="torch.hann_window",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric")
+    ),
+    periodic=st.booleans(),
+    dtype=helpers.get_dtypes("numeric", full=False, none=True),
+    requires_grad=_requires_grad(),
+)
+def test_torch_hann_window(
+    periodic,
+    dtype_and_x,
+    dtype,
+    num_positional_args: 2,
+    on_device,
+    frontend,
+    requires_grad,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
```

Comment:
There seems to be missing kwargs for `test_flags` and `fn_tree`

Reply:
yes, I missed it. i.e: `fn_tree` & `test_flags` need to  pass the generated values to `helpers.test_frontend_function` 

sorry to bother you so much....